src/Airy/airy.jl

```diff
@@ -0,0 +1,299 @@
+#######
+####### Real arguments
+#######
+
+# airyai & airyaix
+
+function airyai(x::Float64)
+    if x >= 2.06
+        return exp(-2 * (sqrt(x)^3 / 3)) * airyaix_large_pos_arg(x)[1]
+    else
+        if x <= -9.5
+            return airyai_large_neg_arg(x)[1]
+        else
+            return airyai_small_arg(x)
+        end
+    end
+end
+
+function airyaix(x::Float64)
+    if x >= 2.06
+        return airyaix_large_pos_arg(x)[1]
+    elseif x >= 0.0
+        # taylor expansion at x = 1.5 
+        b = evalpoly(x - 1.5, (0.07174949700810541, -0.09738201284230132, 0.05381212275607906, -0.012387253709224428, -0.0013886523923485612, 0.0017615621096121207, -0.00048234107659157565, 2.9849780287371904e-5, 1.853661597722781e-5, -6.0773111966738585e-6, 6.406078250357068e-7, 8.5642265292882e-8, -3.876060196303256e-8, 4.929943737019422e-9, 1.5110638652930308e-10, -1.493604112262068e-10, 2.148584715338907e-11, -2.6814055261032026e-13, -3.827831388762196e-13, 6.164805942828536e-14, -2.2166191076964464e-15, -6.912167850804561e-16, 1.2624054278515299e-16, -6.42973178916429e-18, -9.091593675774033e-19, 1.9432657516901093e-19, -1.198995513927753e-20, -8.798710894927164e-22, 2.33256140820231e-22, -1.6391332233394835e-23, -6.091803198418045e-25))
+        return exp(2 * (sqrt(x)^3 / 3)) * b
```

Comment:
```suggestion
return exp((2/3) * x * sqrt(x)) * b
```

Reply:
Ughh yes thank you!... I wonder if that will help improve ULP here. (Scaled versions peak at around <1.9 ULP and regular version grow unbounded for large `x`.